mozci/console/commands/push.py

```diff
@@ -363,10 +395,45 @@ def log_pushes(self, status, backedout):
             ]
         )
         verb = "were" if backedout else "weren't"
-        self.line(
-            f"{nb} out of {len(self.pushes)} pushes {verb} backed-out by a sheriff and were classified as {status.name}."
+        line = f"{nb} out of {len(self.pushes)} pushes {verb} backed-out by a sheriff and were classified as {status.name}."
+        self.line(line)
+
+        return line
+
+    def send_emails(self, total, stats, error_line):
+        notify = taskcluster.Notify(get_taskcluster_options())
+
+        today = datetime.datetime.strftime(datetime.datetime.now(), "%Y-%m-%d")
+        subject = f"Mozci | classify-eval report generated the {today}"
+
+        stats = "\n".join([f"- {stat}" for stat in stats])
+        content = EMAIL_CONTENT.format(
+            today=today,
+            total=total,
+            error_line=f"**{error_line}**" if error_line else "",
+            stats=stats,
         )
 
+        emails = config.get("emails", [])
+        if not emails:
+            self.line(
+                "<info>--send-email option was provided but no email recipient was found in the configuration.</info>"
+            )
+
+        for idx, email in enumerate(emails):
+            try:
+                notify.email(
+                    {
+                        "address": email,
+                        "subject": subject,
+                        "content": content,
+                    }
+                )
+            except Exception:
+                self.line(
+                    f"<error>Failed to send the report by email for the address at index {idx} in the provided list.</error>"
```

Comment:
Add the email + exception content here for debug:

```suggestion
                    f"<error>Failed to send the report by email n°{index} {email} : {e} .</error>"
```